src/main/java/seedu/address/model/appointment/Appointment.java

```diff
@@ -1,37 +1,48 @@
 package seedu.address.model.appointment;
 
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
 import java.time.LocalDate;
 import java.time.LocalTime;
 import java.time.format.DateTimeFormatter;
-import java.time.temporal.ChronoField;
+import java.util.Locale;
 
+import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.model.person.Name;
 
 /**
- * Represents an appointment with a person in SocialBook.
+ * Represents an appointment with a person in the address book.
  * Guarantees: immutable; field values are validated
  */
 public record Appointment(Name name, LocalDate date, LocalTime startTime, LocalTime endTime) {
 
+    public static final String MESSAGE_INVALID_TIME_INTERVAL = "Appointment start time must be before end time";
+
+    public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("EEEE, MMMM d, yyyy");
+    public static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("h:mm a", Locale.US);
```

Comment:
Would there be any difference if `new Locale("en", "SG")` is used instead of `Locale.US`?

Reply:
I think `Locale.US` enforces uppercase `AM/PM`, but I'm not sure about `Locale.SG`.